src/emulator/eventarcEmulator.ts

```diff
@@ -0,0 +1,169 @@
+import * as _ from "lodash";
+import * as express from "express";
+import * as api from "../api";
+import { logger } from "../logger";
+import { Constants } from "./constants";
+import { EmulatorInfo, EmulatorInstance, Emulators } from "./types";
+import { createDestroyer } from "../utils";
+import { EmulatorLogger } from "./emulatorLogger";
+import { EventTrigger } from "./functionsEmulatorShared";
+import { CloudEvent } from "./events/types";
+import { EmulatorRegistry } from "./registry";
+
+interface CustomEventTrigger {
+  projectId: string;
+  triggerName: string;
+  eventTrigger: EventTrigger;
+}
+
+interface RequestWithRawBody extends express.Request {
```

Comment:
In firebase-functions, we have this exact same type and we call it Request. For consistency, can we just call this Request (or import the firebase-functions type?)

Reply:
I believe other emulators (e.g. Functions emulator) also call it RequestWithRawBody which is what this is consistent with. What do you think?

Reply:
Ahh, good point - you're right that its better to be consistent within this repo than with firebase-functions. Good call.